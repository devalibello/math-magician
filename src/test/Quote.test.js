import React from 'react';
import { render } from '@testing-library/react';
import Quote from '../components/Quote';

describe('Quote', () => {
  test('should render the Quote component', () => {
    const { container } = render(<Quote />);
    expect(container.firstChild).toMatchSnapshot();
  });
});
