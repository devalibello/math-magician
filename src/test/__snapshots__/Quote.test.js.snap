// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Calculator should render the Calculator component 1`] = `
<div
  class="apiQuote"
  data-testid="test-quote"
>
  <p>
    Quote Loading...
  </p>
</div>
`;

exports[`Quote should render the Quote component 1`] = `
<div
  class="apiQuote"
  data-testid="test-quote"
>
  <p>
    Quote Loading...
  </p>
</div>
`;
